app/views/index.scala.html

```diff
@@ -24,6 +28,15 @@
             background-color: rgba(192, 190, 207, 0.4);
             left: 0px;">
         </div>
+
+@*
+        <a href="@newsRibbonLink" class="ps-news-ribbon" title="News on PS">News on PS</a>
+*@
+
+        @if(newsRibbonLink != null && newsRibbonLink.exists(_.trim.nonEmpty)) {
+            <a href="@newsRibbonLink" class="ps-news-ribbon" title="News on PS">News on PS</a>
```

Comment:
I don't think we need the "title" attribute, that just adds a tooltip on hover, right? Seems distracting more than anything!